internal/framework/provider/appsv1/validating_admission_policy_schema_gen.go

```diff
@@ -0,0 +1,314 @@
+						Optional: true,
+					},
+					"generation": schema.Int64Attribute{
+						MarkdownDescription: `A sequence number representing a specific generation of the desired state. Populated by the system. Read-only.`,
+						Optional:            true,
+						Computed:            true,
+					},
+					"labels": schema.MapAttribute{
+						MarkdownDescription: `Map of string keys and values that can be used to organize and categorize (scope and select) objects. May match selectors of replication controllers and services. More info: https://kubernetes.io/docs/concepts/overview/working-with-objects/labels`,
+						ElementType:         types.StringType,
+						Optional:            true,
+					},
+					"name": schema.StringAttribute{
+						MarkdownDescription: `Name must be unique within a namespace. Is required when creating resources, although some resources may allow a client to request the generation of an appropriate name automatically. Name is primarily intended for creation idempotence and configuration definition. Cannot be updated. More info: https://kubernetes.io/docs/concepts/overview/working-with-objects/names#names`,
+						Optional:            true,
+						Computed:            true,
+					},
+					"namespace": schema.StringAttribute{
+						MarkdownDescription: `Namespace defines the space within which each name must be unique. An empty namespace is equivalent to the "default" namespace, but "default" is the canonical representation. Not all objects are required to be scoped to a namespace - the value of this field for those objects will be empty.
+
+Must be a DNS_LABEL. Cannot be updated. More info: https://kubernetes.io/docs/concepts/overview/working-with-objects/namespaces`,
+						Optional: true,
+					},
+					"resource_version": schema.StringAttribute{
+						MarkdownDescription: `An opaque value that represents the internal version of this object that can be used by clients to determine when objects have changed. May be used for optimistic concurrency, change detection, and the watch operation on a resource or set of resources. Clients must treat these values as opaque and passed unmodified back to the server. They may only be valid for a particular resource or set of resources.
+
+Populated by the system. Read-only. Value must be treated as opaque by clients and . More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#concurrency-control-and-consistency`,
+						Optional: true,
+						Computed: true,
+					},
+					"uid": schema.StringAttribute{
+						MarkdownDescription: `UID is the unique in time and space value for this object. It is typically generated by the server on successful creation of a resource and is not allowed to change on PUT operations.
+
+Populated by the system. Read-only. More info: https://kubernetes.io/docs/concepts/overview/working-with-objects/names#uids`,
+						Optional: true,
+						Computed: true,
+					},
+				},
+			},
+			"spec": schema.SingleNestedAttribute{
```

Comment:
this looks to be good, I did notice that it's missing the `status` block, this can be seen from the api reference: https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.31/#validatingadmissionpolicy-v1-admissionregistration-k8s-io

Did you happen to use the code generation tool for this? Be sure to remove `_gen` from the end of all the files if you didn't use the generation tool.

Reply:
I've renamed the files and I've added the status field!